src/index.html

```diff
@@ -20,11 +20,15 @@
   </head>
   <body>
     <header class="header">
+      <a
+        class="header__logo-link"
+        href="./index.html"
+      >
       <img
         src="images/logo.png"
-        class="header__logo"
         alt="logo"
       >
```

Comment:
Let it pass, I did indents here but didn't push it

Reply:
If it were just indents, I would appove the task 😄 
Check-list for each task is different.
In this item 3 reads:
> logo should be a link to the main page of the website